specification/1.0/README.md

```diff
@@ -163,14 +159,11 @@ Examples:
 }
 ```
 
-
 ## Accessors
 
 Accessors reference individual objects inside bufferviews. What they are referencing depends on the type of bufferview. In some cases they reference a complete bufferview.
 
-Examples: 
-
-### Geometric object in a 3dm file
+### Example: Geometric object in a 3dm file
 
```

Comment:
A short description would be nice here as well.
Something like, "Referencing an individual object located in [bufferViews[0]](#reference-buffer) via its id".

Reply:
Fully agreed, could you do it please?